### ISSUE TITLE
Any way to disable auto-fetch/refresh translations when using $translate.use() setter?

### ISSUE THREAD

**Issue by jfcere** [angular-translate/angular-translate#1500]
Hi there, I posted a question on StackOverflow like 2 weeks ago but without success so I'm gonna take a chance here.

[Is it possible to not refresh/fetch translations automatically when using $translate.use() setter](http://stackoverflow.com/questions/37167899/is-it-possible-to-not-refresh-fetch-translations-automatically-when-using-trans)
### TL:DR

Using an MVC API that already returns translated strings according to the Accept-Language property in the Request Headers I am forcing an application reload when user is changing language which make the auto-fetch/refresh translation filters useless in my case.

Although everything works fine the only drawback that I am not really happy with is that using $translate.use() setter fetch the JSON file and automatically refresh the translations which is not needed since we force the application to reload anyway.
### Question

So basically ... is there any way to set the language to Angular-Translate without auto fetching/refreshing translations ?
### My environment
- version of angular-translate : 2.11
- version of angular : 1.5.5
- which browser and its version : Chrome 50.0.2661.102

**Comment by knalli** [comment]
Ehm.. why do you call `$translate.use()` anyway? The `location.reload()` should be sufficient, shouldn't it?

**Comment by jfcere** [comment]
Thx for your reply and your time, this is really appreciated!

Following the documentation [here](https://angular-translate.github.io/docs/#/api/pascalprecht.translate.$translate) 

> **use([key])**
Tells angular-translate which language to use by given language key. This method is used to change language at runtime. It also takes care of storing the language key in a configured store to let your app remember the choosed language.

I am currently using ``$translateProvider.useLocalStorage()`` which sets a key with specific name in the local storage. If I don't use ``$translate.use()`` when the language is changed, the local storage won't be updated (unless I do it manually and that means I would have to hard-code the same key name as Angular-Translate uses and who knows if it would ever change). That means that when the application will reload it will use the non-updated key value and therefore won't change the language in the application.

So basically I was looking for a "built-in" way to tell Angular-Translate to use a new language (change the language in local storage) without fetching/refreshing automatically the translations.

I'm I missing something ?

Thanks again!

**Comment by knalli** [comment]
I don't understand the problem. If you use `$translate.use()`, it will switch the language and store this information.

Perhaps try to minimize your demo code and provide a running example.

**Comment by jfcere** [comment]
@knalli I agree that it might be complicated to understand with all the provided code.

I'll make a running example and link it back here soon.

**Comment by knalli** [comment]
Hello,

I'm closing this issue due inactivity. We have asked for additional details, but you could not helped us. Therefore we close this issue as not reproducible.

If you still have an issue, please create a new issue because we cannot be sure to track closed issues.
